### ISSUE TITLE
Feature Request: Option to keep the URLs short visually

### ISSUE THREAD

**Issue by chromaniac2** [polomarcus/faster-links#9]
Basically, just change the actual link not the visible link. TweetDeck looks pretty messy with tweets with insanely long links that websites tend to have these days! Thanks for this great app. 
![chrome_2017-03-27_21-37-41](https://cloud.githubusercontent.com/assets/310098/24366077/a4003fea-1335-11e7-9cf6-11d226da5e14.png)

**Comment by polomarcus** [comment]
Hi @Sushubh ,

Yeah, they love to do so for SEO... haha

It's true that it looks pretty messy :sweat_smile: , What do you think to shorten the URL, by keeping only the X chars of it, if it gets too long ? 

Thanks for the feedback!

**Comment by chromaniac2** [comment]
Yup. That would be nice as well. Full link can be viewed in the status bar or tooltip in any case! Thanks.
